src/confighub/components/Header.tsx

```diff
@@ -38,7 +38,7 @@ const Header = ({ title, status, onSaveClick }: IHeaderProps) => {
       </HeaderTitleArea>
       <HeaderSideContainer>
         <Button
-          text='Save configuration'
+          text='Save configuration Test'
```

Comment:
Probably there shouldnt be "test" here ?

Reply:
done